README.md

```diff
@@ -1 +1,259 @@
-# Hello-Microverse-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
[OPTIONAL] Good looking README ✨ but would look even better if you remove the Microverse logo or replace it with yours. 😉